doc/data/messages/c/consider-using-generator/details.rst

```diff
@@ -0,0 +1,6 @@
+Removing ``[ ]`` inside calls that can use containers or generator indistinctly should be considered for performance reasons since
+list comprehension will make unnecessary data copies. The performance will be better if
+you are working with really long lists or sets.
+
+Regarding performance it's the same for ``max``, ``min`` and ``sum``: you should consider
+your dataset and your environment. But it's also best practice recommended by pep289.
```

Comment:
```suggestion
Removing ``[]`` inside calls that can use containers or generator indistinctly should be considered for performance reasons since a generator will have an upfront cost to pay. The performance will be better if
you are working with really long lists or sets and depending on the interpreter: you should consider
your dataset and your environment. 

For ``max``, ``min`` and ``sum` using a generator is also best practices recommended by pep289.
```